_includes/tutorials/deserialization-errors/ksql/markup/dev/check-errors-query.adoc

```diff
@@ -2,10 +2,17 @@ Now here comes the fun part. We know that at least one of the records produced h
 Therefore, this one error needs to show up somewhere.
 With the https://docs.confluent.io/current/ksql/docs/developer-guide/processing-log.html[KSQL Processing Log] feature enabled, you can query a stream called `KSQL_PROCESSING_LOG` to check for deserialization errors.
 
+The query below is extracting some of the data available in the processing log.
+As we configured the processing log to include the payload of the message, we can also use the `encode` method to convert the record from `base64` encoded into a human readable `utf8` encoding:
+
 +++++
 <pre class="snippet"><code class="sql">{% include_raw tutorials/deserialization-errors/ksql/code/tutorial-steps/dev/check-errors-query.sql %}</code></pre>
 +++++
 
+Notice we needed to quote the `topic` field in the `WHERE` clause, as its a reserved word.
+
+While the `ERRORMESSAGE` is a little cryptic in this instances, the `CAUSE` and `MSG` columns would be enough to diagnose the issue here.
```

Comment:
```suggestion
While the `ERRORMESSAGE` is a little cryptic in these instances, the `CAUSE` and `MSG` columns would be enough to diagnose the issue here.
```

Reply:
There's only a single error in the example, so went with `this instance`.